validation/static/index.js

```diff
@@ -25,8 +25,36 @@ document.addEventListener('DOMContentLoaded', () => {
 
                 if (judgement === 'yes') {
                     button.setAttribute("class", "button button--success-solid translation-judgement-accuracy-yes")
+                    
+                    const noButtons =  document.getElementsByClassName("translation-judgement-accuracy-no");
+                    for (let b of noButtons) {
+                        if (b.dataset.phraseId === phraseId) {
+                            b.setAttribute("class", "button button--fail translation-judgement-accuracy-no")
+                        }
+                    }
+
+                    const headers = document.getElementsByClassName("card__header")
+                    for (let h of headers) {
+                        if (h.dataset.phraseId === phraseId) {
+                            h.setAttribute("class", "card__header card__header--green")
```

Comment:
Again, with `element.classList`:

```suggestion
                            h.classList.add("card__header--green")
```